server/config/development.json

```diff
@@ -0,0 +1,5 @@
+{
+    "uptimeRobot": {
+		"iplanHeartbeatUrl": "https://www.google.com"			
```

Comment:
you could also make it "" in dev and add code that checks if anything is there, and does not try to hit any URLs if the string is empty <3

Reply:
i didnt want code that isnt called, seems better to have it tested for real